### PR TITLE
Add surfer

### PR DESCRIPTION
**Pull request by nebulade**
Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [x] Your submissions are formatted according to the guidelines. 
        
    ``[Name](http://homepage/) - Short description. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code)) `License` `Language```
- [x] Your additions are ordered alphabetically.
- [x] Your additions are [free software](https://en.wikipedia.org/wiki/Free_software), or if not they have been added to [non-free](non-free.md).
- [x Any licenses you have added are in our [list of licenses](https://github.com/Kickball/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [x] You have searched the repository for any relevant issues or PRs.
- [x] Any category you are creating has the minimum requirement of 3 items.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -727,6 +727,7 @@ See also <https://en.wikipedia.org/wiki/List_of_streaming_media_systems>, <https
   * [Mindmaps](https://github.com/drichard/mindmaps) - An open source, offline capable, mind mapping application. ([Demo](http://drichard.org/mindmaps/)) `AGPLv3` `HTML5`
   * [Musical Artifacts](https://musical-artifacts.com/) - Helping to catalog, preserve and free the artifacts you need to produce music. ([Source Code](https://github.com/lfzawacki/musical-artifacts)) `MIT` `Ruby`
   * [Skygear](https://skygear.io) - A complete backend server for web, mobile and iOT applcations. ([Demo](https://portal.skygear.io), [Source Code](https://github.com/SkygearIO/skygear-server)) `Apache` `Go`
+  * [Surfer](https://github.com/nebulade/surfer) - Simple static file server with webui to manage files.  `MIT` `Nodejs`
```
Comment (AndyR207) [inline README.md]: You have a double space between the description and the license. Please correct this.

**Comment by nebulade** [comment]
Sorry about the whitespace, I used the Makefile add this time and didn't realize it does not trim.

**Comment by nodiscc** [comment]
Wouldn't the [web based file managers](https://github.com/Kickball/awesome-selfhosted#web-based-file-managers) section be a better fit?

**Comment by nebulade** [comment]
Indeed, apparently DirectoryLister for example also serves up files, just like this app. So the web base file managers does apply.

**Pull request merged**